backend/app/src/configs/chat-event.constants.ts

```diff
@@ -1,7 +1,10 @@
 export const chatEvent = {
   JOIN: 'JOIN',
   LEAVE: 'LEAVE',
+  CREATE: 'CREATE',
   SEND: 'SEND',
   RECEIVE: 'RECEIVE',
   NOTICE: 'NOTICE',
+  ADD_ADMIN: 'ADD_ADMIN',
+  REMOVE_ADMIN: 'REMOVE_ADMIN',
 }
```

Comment:
```suggestion
} as const
```
타입 단언 (type assertion) 을 쓰면 더욱 좋을 것 같아요 [(참고링크)](https://blog.toycrane.xyz/typescript%EC%97%90%EC%84%9C-%ED%9A%A8%EA%B3%BC%EC%A0%81%EC%9C%BC%EB%A1%9C-%EC%83%81%EC%88%98-%EA%B4%80%EB%A6%AC%ED%95%98%EA%B8%B0-e926db079f9)